src/App.js

```diff
@@ -15,10 +32,14 @@ function App(): React.Component {
         <p >SEB Building</p>
         </header>
       </div>
-	  <div>
```

Comment:
I think this part is the embedded grafana channel. We should keep it.

Reply:
duplicate lines of code. It appears again after the heat map tags. Line 28-32 (or 49-53)